src/framework/constraints.ml

```diff
@@ -925,7 +927,7 @@ end
 
 (** Transforms a [GenericEqBoxIncrSolver] into a [GenericGlobSolver]. *)
 module GlobSolverFromEqSolver (Sol:GenericEqBoxIncrSolverBase)
-  : GenericGlobSolver
+  (* : GenericGlobSolver *)
```

Comment:
Is the result of the functor application still a `GenericGlobSolver`? If not, one may remove the comment (also above)

Reply:
It should still be, but I think I removed the signature constraint to expose some internals. I'll check if it's still necessary.

Reply:
I removed it altogether. It's not really possible to add a `with type marshal` constraint to this because it's a functor and the actual definition as `Sol'.marshal` depends on multiple functor applications in the functor body.